index.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+require("body-parser");
```

Comment:
No need of `body-parser` if you are using `express.json()` to accept `req.body`!
Please have a look into that :)

Reply:
Thanks for letting me know about this, was not aware!
Just read that express 4.16+ does not need `body-parser` package and use in place of that `express.json()`